api/auth/pg-jwt.js

```diff
@@ -1,19 +1,18 @@
 var jwt = require('jsonwebtoken');
-var toID = require('postgraphql/dist/graphql/types').toID
 
 module.exports = jwtSecret => (req, res, next) => {
     if (!req.user) return next();
 
     var token = jwt.sign({
         user_rowId: req.user.id,
```

Comment:
We should completely remove the id/rowId duplicates. Either use UUID or IDs everywhere